test/integration/src/test/java/io/pravega/test/integration/controller/server/EventProcessorTest.java

```diff
@@ -94,8 +93,12 @@
     private ServiceBuilder serviceBuilder;
```

Comment:
serviceBuilder needs to be closed in the @After method.

Reply:
Good one. Fixed.